packages/@webex/plugin-meetings/test/unit/spec/meeting/utils.js

```diff
@@ -138,12 +138,13 @@ describe('plugin-meetings', () => {
         };
 
         MeetingUtil.parseLocusJoin = sinon.stub();
-        await MeetingUtil.joinMeeting(meeting, {});
+        await MeetingUtil.joinMeeting(meeting, {alias: 'userName'});
```

Comment:
nitpick: This alias could also be `assigned name` to be uniform and be self-explanatory

Reply:
@mkesavan13 Updated the code please do have the re-review